sdk/formrecognizer/ai-form-recognizer/test/public/node/analysis.spec.ts

```diff
@@ -0,0 +1,746 @@
+        const [table] = tables;
+        assert.ok(table.boundingRegions?.[0].boundingBox);
+        assert.equal(table.boundingRegions?.[0].pageNumber, 1);
+      });
+
+      it("pdf file stream without passing content type", async () => {
+        const filePath = path.join(ASSET_PATH, "forms", "Invoice_1.pdf");
+        const stream = fs.createReadStream(filePath);
+
+        const poller = await client.beginExtractLayout(stream, testPollingOptions);
+        const { pages, tables } = await poller.pollUntilDone();
+
+        assert.isNotEmpty(pages);
+
+        assert.isNotEmpty(tables);
+        const [table] = tables;
+        assert.ok(table.boundingRegions?.[0].boundingBox);
+        assert.equal(table.boundingRegions?.[0].pageNumber, 1);
+      });
+
+      it("url", async () => {
+        const url = makeTestUrl("/Invoice_1.pdf");
+
+        const poller = await client.beginExtractLayout(url, testPollingOptions);
+        const { pages, tables } = await poller.pollUntilDone();
+
+        assert.isNotEmpty(pages);
+
+        assert.isNotEmpty(tables);
+        const [table] = tables;
+        assert.ok(table.boundingRegions?.[0].boundingBox);
+        assert.equal(table.boundingRegions?.[0].pageNumber, 1);
+      });
+
+      it("with selection marks", async () => {
+        const filePath = path.join(ASSET_PATH, "forms", "selection_mark_form.pdf");
+        const stream = fs.createReadStream(filePath);
+
+        const poller = await client.beginExtractLayout(stream, testPollingOptions);
+
+        const { pages } = await poller.pollUntilDone();
+
+        assert.isNotEmpty(pages);
+
+        /* There should be a table on the page, but the layout engine does not recognize it, maybe because it is too small and sparse.
+          assert.isNotEmpty(tables);
+          const [table] = tables;
+          assert.ok(table.boundingRegions?.[0].boundingBox);
+          assert.equal(table.boundingRegions?.[0].pageNumber, 1);*/
+
+        assert.equal(pages[0].pageNumber, 1);
+        assert.isNotEmpty(pages[0].selectionMarks);
+      });
+
+      it("specifying locale", async () => {
+        const url = makeTestUrl("/Invoice_1.pdf");
+
+        // Just make sure that this doesn't throw
+        const poller = await client.beginExtractLayout(url, {
+          locale: "en-US",
+          ...testPollingOptions,
+        });
+
+        await poller.pollUntilDone();
+      });
+
+      it("invalid locale throws", async () => {
+        const url = makeTestUrl("/Invoice_1.pdf");
+
+        try {
+          const poller = await client.beginExtractLayout(url, {
+            locale: "thisIsNotAValidLanguage",
+            ...testPollingOptions,
+          });
+
+          await poller.pollUntilDone();
+          assert.fail("Expected an exception due to invalid language.");
+        } catch (ex) {
+          // Just make sure we didn't get a bad error message
+          assert.isFalse((ex as Error).message.includes("<empty>"));
+        }
```

Comment:
I think this could be rewritten more concisely without the try/catch using `await assert.isRejected(poller.pollUntilDone(), /<empty>/)`.